app/src/main/java/com/edulectronics/tinycircuit/Controllers/WireController.java

```diff
@@ -118,4 +121,46 @@ public Line getVerticalWire(Point a, Point b) {
             return null;
         return new Line(new Point(a.x, a.y), new Point(a.x, b.y - (Math.abs(a.y - b.y) % 150)));
     }
+
+    public Line getEndPointLine(Point startPoint, Point endPoint, ConnectionPointOrientation orientation) {
```

Comment:
Andere naam lijkt me beter. Nu gebruik je hem voor startPoint en EndPoint. Misschien is iets als getHalfLine een beter idee?